### DISCUSSION TITLE
Unknown Attribute 'Image'

### DISCUSSION THREAD

**Discussion by j2yau** [motor-admin/motor-admin#118] [Q&A]
Hi,

I created a new Column for Image and tried to upload a jpeg file through the form. However, it gives an error "unknown attribute 'image' for Product."
![Screenshot 2023-09-30 163533](https://github.com/motor-admin/motor-admin/assets/140708598/76bb1bf8-0dcf-45ae-af6c-8e5b0abd3042)

**Comment by omohokcoj** [comment]
@j2yau instead - please try to create reference column with attachment, like described here:
https://docs.getmotoradmin.com/guide/file_attachments.html
